articles/file-handling-in-Cpp/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/file-handling-in-Cpp/
+title: File Handling in C++
+description: In this article, we will walk through how to implement the File Handling function in C++, and we will cover the key working pointers of file handling.
+author: parampreet-singh
+date: 2020-09-21T00:00:00-12:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/file-handling-in-Cpp/hero.jpg
+    alt: functions Function C++ cpp
+---
+If you are a Software Engineer, you may come across File handling at some point in your career. It is the ability to store the output and perform various operations on it, such as updating the data or overwriting things in a file. Files are always a vital way to store data.
+<!--more-->
+### Why do you need File Handling?
+- **Reusability**: Generally, developers work with big data, and it's always not efficient to write everything out. But by storing data in files, one can use it anytime.
+
+- **Larger Storage Capacity**: Storing a large amount of data or test cases (to make sure that your code runs in every condition) is a tedious task, and by using files, a developer can store that data easily.
+
+- **Portability**: With files, developers can transfer the data without worrying about data loss.
+
+In this article, we will walk through how to implement the File Handling function in C++, and we will cover the key working pointers of File handling such as:
+
+- Opening a File
+- Writing to a File
+- Reading from a File
+- Closing a file
+
+To perform File Handling functions, we need to add another library; fstream. The library holds all the functions related to File Handling. Here's how to include it.
```

Comment:
perform File Handling functions -> perform file handling functions

Reply:
add another library; fstream. The library -> add another library: fstream. The library

Reply:
fstream. The library holds -> fstream. This library holds

Reply:
related to File Handling. Here's -> related to handling files. Here's